### PR TITLE
date_format min and max date tests

### PR DESCRIPTION
**Pull request by Arthasasyan**
Tests for issue 6416

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/function/date/DateFormatMinAndMaxDateTest.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    @Test
+    @Issue("6416")
+    public void testMaxDate() {
+        String sqlQuery = String.format("SELECT date_format(max(time)) FROM \"%s\" WHERE datetime BETWEEN '%s' AND '%s'" +
+                        " GROUP BY PERIOD(1 day)", METRIC,
+                Util.MIN_STORABLE_DATE, Util.MAX_STORABLE_DATE);
+        String[][] expectedRow = {
+                {STORED_TIME}
+        };
+        assertSqlQueryRows("Min date is incorrect for date_format function", expectedRow, sqlQuery);
```
Comment (raipc) [inline src/test/java/com/axibase/tsd/api/method/sql/function/date/DateFormatMinAndMaxDateTest.java]: Max

File: src/test/java/com/axibase/tsd/api/method/sql/function/date/DateFormatMinAndMaxDateTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    @BeforeClass
+    public void prepareData() throws Exception{
+        Series series = new Series(ENTITY, METRIC);
+        series.addSamples(Sample.ofDateInteger(STORED_TIME, VALUE));
+        SeriesMethod.insertSeriesCheck(series);
+    }
+
+    @Test
+    @Issue("6416")
+    public void testMinDate() {
```
Comment (raipc) [inline src/test/java/com/axibase/tsd/api/method/sql/function/date/DateFormatMinAndMaxDateTest.java]: Add description of the original issue

Comment (Arthasasyan) [reply src/test/java/com/axibase/tsd/api/method/sql/function/date/DateFormatMinAndMaxDateTest.java]: Should it be added as Test description?

Comment (raipc) [reply src/test/java/com/axibase/tsd/api/method/sql/function/date/DateFormatMinAndMaxDateTest.java]: yes

**Pull request merged**